src/MUIDataTable.js

```diff
@@ -317,6 +318,9 @@ class MUIDataTable extends React.Component {
     if (props.options.responsive === 'scroll') {
       console.error('This option has been deprecated. It is being replaced by scrollMaxHeight');
     }
+    if (props.options.textLabels) {
```

Comment:
I'm not sure we need this as it looks like a holdover from a previous fix, but we're merging in the default `textLabels` in the props `textLabels` elsewhere now.